src/CommunityToolkit.Maui/Platforms/Android/Views/SnackBar.cs

```diff
@@ -0,0 +1,156 @@
+﻿#if ANDROID
```

Comment:
1) Since this file is under `Platforms/Android`, it shouldn't need `#if ANDROID` (if I understand MAUI correctly)
2) Is XCT gonna use `Platforms` folder to separate code for different platforms instead of doing that based on a file name? I liked the original approach where all code for a feature was in the same place.

CC: @jfversluis

Reply:
1. I expected the same, but it doesn't work on my device. Maybe because this is a Class library project and not a Console application.
2. I don't think we should change it in XCT. As for Maui separation by platforms folders out of the box it make sense here.